htdocs/comm/propal/class/propal.class.php

```diff
@@ -1455,6 +1455,12 @@ public function createFromClone(User $user, $socid = 0, $forceentity = null, $up
 							$line->subprice = $pu_ht;
 							$line->tva_tx = $tva_tx;
 							$line->remise_percent = $remise_percent;
+							// Update also cost price
+							$subpricetmp = isset($conf->global->ForceBuyingPriceIfNull) && $conf->global->ForceBuyingPriceIfNull > 0 ? 0 : $line->subprice;
+							$result = $object->defineBuyPrice($subpricetmp, $line->remise_percent, $line->fk_product);
```

Comment:
We should never have data that is silently updated into database without the consent of the user.